src/codewhisperer/activation.ts

```diff
@@ -262,7 +262,9 @@ export async function activate(context: ExtContext): Promise<void> {
         updateCloud9TreeNodes.execute()
     } else if (isInlineCompletionEnabled()) {
         await setSubscriptionsforInlineCompletion()
-        await vscode.commands.executeCommand('setContext', 'CODEWHISPERER_ENABLED', true)
+        if (await getManualTriggerStatus()) {
+            await vscode.commands.executeCommand('setContext', 'CODEWHISPERER_ENABLED', true)
+        }
```

Comment:
I'd remove the if statement as it introduces a bug.

Reply:
Or you can move the context command somewhere else. Is there a single method for setting/checking CodeWhisperer enablement?